src/platform/utilities/demo-script/local-demo.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable no-console */
+const commandLineArgs = require('command-line-args');
+
+const args = commandLineArgs([{ name: 'from', type: Number }]);
```

Comment:
Is this script supposed to be here or is it just for testing?

Reply:
This is just for testing/demoing.